src/pages/Dashboard/Dashboard.component.tsx

```diff
@@ -0,0 +1,85 @@
+import { Link } from 'react-router-dom';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Container from '@mui/material/Container';
+import Typography from '@mui/material/Typography';
+
+import { Urls } from '../../constants';
+import { PageAligner } from '../../components/PageView/PageView.component';
+
+import { TransactionsTableContainer } from './TransactionsTable/TransactionsTable.container';
+import { BalanceBlockProps, DashboardComponentProps } from './Dashboard.types';
+
+const BalanceBlock = ({ amount, label }: BalanceBlockProps) => (
+  <Container sx={{ padding: 2 }}>
+    <Typography variant="body2" align="center">
+      {label}
+    </Typography>
+    <Typography variant="h2" align="center" mt={1}>
+      {amount} FISH
+    </Typography>
+  </Container>
+);
+
+export const DashboardComponent = ({
+  totalUSD,
+  fishBalance,
+  fishVesting,
+}: DashboardComponentProps) => (
+  <PageAligner>
+    <Box
+      sx={{
+        gap: 2,
+        width: '90%',
+        display: 'flex',
+        maxWidth: 1400,
```

Comment:
Let's maybe extract it to some king of constant? We'll probably use it a bit

Reply:
Dashboard has slightly different styles than other pages, I don't think it's worth extracting this for now